src/UserStories/UserStories.css

```
@@ -8,31 +8,36 @@ html {
   overflow: hidden;
 }
 .stories-scrollable {
-  display: flex;
-  padding-left: 50vw;
-  transition: transform 0.5s ease-in-out;
+  transition: transform 0.2s ease-out;
   height: 100vh;
```

I'd make this 
```suggestion
  min-height: 100vh;
```

To show a vertical scroll when the window is wide and short.

---

Done